tests/unit/flow/test_flow.py

```diff
@@ -409,9 +409,9 @@ def validate(req):
     f = Flow(restful=restful, read_only=True).add(
         uses=os.path.join(cur_dir, '../yaml/datauriindex.yml'), timeout_ready=-1
     )
-
+    files = os.path.join(cur_dir, '*.py')
```

Comment:
interesting that we use python files as text input.
Wouldn't it be better to create files in the tmpdir instead of reading from the program code

Reply:
We can just use the yaml files. We know these will always have content